src/content/en/2022/security.md

```diff
@@ -1,22 +1,1281 @@
 title: Security
-#TODO - Review and update chapter description
 description: Security chapter of the 2022 Web Almanac covering Transport Layer Security, content inclusion (CSP, Feature Policy, SRI), web defense mechanisms (tackling XSS, XS-Leaks), and drivers of security mechanism adoptions.
-authors: [SaptakS, feross, lirantal]
-reviewers: [kushaldas, tomvangoethem, nrllh, clarkio]
+authors: [SaptakS, lirantal, clarkio]
+reviewers: [kushaldas, tunetheweb]
 analysts: [VictorLeP, vikvanderlinden, GJFR]
-editors: []
+editors: [tunetheweb]
 translators: []
+SaptakS_bio: Saptak S is a human rights centered web developer, focusing on usability, security, privacy and accessibility topics in web development. He is a contributor and maintainer of various different open source projects like <a hreflang="en" href="https://www.a11yproject.com">The A11Y Project</a>, <a hreflang="en" href="https://onionshare.org/">OnionShare</a> and <a hreflang="en" href="https://wagtail.org/">Wagtail</a>. You can find him blogging at <a hreflang="en" href="https://saptaks.blog">saptaks.blog</a>.
+lirantal_bio: Known for his open source and JavaScript security initiatives, Liran Tal is an award-winning software developer, security researcher, and open source champion in the JavaScript community. He's an internationally recognized GitHub Star, acknowledged for his open source advocacy, and has received the OpenJS Foundation's Pathfinder for Security for his work on Node.js security. His contributions to developer security education include leading OWASP projects, building supply chain security tools, participation in CNCF and OpenSSF initiatives, and authoring books such as O'Reilly's Serverless Security. He leads the developer advocacy team at Snyk.io and is on a mission to empower developers with better application security skills.
+clarkio_bio: Brian is a developer who has been building and architecting software for all different types of purposes and applications. His initial focus was on <a hreflang="en" href="https://www.asp.net/">ASP.NET</a>, <a hreflang="en" href="https://www.microsoft.com/net/">C#/.NET</a>, SQL and WPF, but he has since shifted to technologies such as <a hreflang="en" href="https://swift.org/getting-started/">Swift</a>, <a hreflang="en" href="https://ionicframework.com/">Ionic</a>, <a hreflang="en" href="https://angular.io/">Angular</a> and <a hreflang="en" href="https://nodejs.org">Node.js</a> (basically JavaScript). Brian is an <a hreflang="en" href="https://www.pluralsight.com/authors/brian-clark">author of several courses</a> on <a hreflang="en" href="https://www.pluralsight.com/">Pluralsight</a> and co-authored the <a hreflang="en" href="https://angular.io/docs/ts/latest/guide/security.html">Angular Security Guide</a> documentation. He has presented many talks at public and private conferences as well as at local code camps covering topics such as secure application development practices and getting started in Open Source Software.
```

Comment:
@clarkio I stole this from your website. I'm going to merge this soon but feel free to open a PR to change this is you want something else.

Reply:
@tunetheweb thanks for grabbing it from there. I need to update my website with a new bio. Could I share the update here with you? Or how should I proceed to open a PR here?

Reply:
Yup PR would be good. You can just find the `src/content/en/security.md` file and edit it right here on GitHub.com for a simple edit like this.

Reply:
Ok got it, thanks. Here's the PR: https://github.com/HTTPArchive/almanac.httparchive.org/pull/3160